src/auth.ts

```diff
@@ -88,7 +89,59 @@ export async function ownerMiddleware(req, res, next) {
     return
   }
 
-  const token = req.headers['x-user-token'] || req.cookies['x-user-token']
+  // Here we are grabing both the x-user-token and x-transport-token
+  const x_user_token =
+    req.headers['x-user-token'] || req.cookies['x-user-token']
+  const x_transport_token =
+    req.headers['x-transport-token'] || req.cookies['x-transport-token']
+
+  console.log('Transport toke:', x_transport_token)
+
+  // default assign token to x-user-token
+  let token = x_user_token
+
+  // Read the transport private key since we will need to decrypt with this
+  const transportPrivateKey = fs.readFileSync('transportPrivate.pem')
+
+  // If we see the user using the new x_transport_token
+  // we will enter this if block and execute this logic
+  if (x_transport_token) {
+    // Decrypt the token and split by space not sure what
+    // the correct way to do the delimiting so I just put
+    // a space for now
+    const splitTransportToken = rsa
+      .decrypt(transportPrivateKey, x_transport_token)
+      .split(' ')
```

Comment:
Right now I'm delimiting the encrypted x-user-token and timestamp with a space is there a better character to use or way to do this?

Reply:
if you use UNIX timestamp its always 10 digits, so you can just pop off the last 10 characters. But not a big deal, space is probably fine

Reply:
ok sounds good I can change to UNIX timestamp and then just pop off the last 10 characters so then we don't need to worry about a delimeter